src/clr-angular/emphasis/alert/providers/multi-alert.service.ts

```diff
@@ -4,14 +4,15 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import { Injectable, QueryList } from '@angular/core';
-import { Observable } from 'rxjs';
+import { Observable, Subscription } from 'rxjs';
 import { Subject } from 'rxjs';
 
 import { ClrAlert } from '../alert';
 
 @Injectable()
 export class MultiAlertService {
   private allAlerts: QueryList<ClrAlert> = new QueryList<ClrAlert>();
+  private _subscription: Subscription;
```

Comment:
### A violation of our coding guidelines. ☠️ 

😂 Was there a specific reason you need to prefix these properties with underscores

Reply:
😭